Makefile

```diff
@@ -395,29 +395,6 @@ clean:
 genconfigdocs:
 	cd cmd/configschema && $(GOCMD) run ./docsgen all
 
-.PHONY: generate-all-labels
-generate-all-labels:
```

Comment:
Can we keep an option to do this via the makefile? We don't have any other situations (I think) where we'd manually execute a script. Rather keep everything accessible via the makefile if possible. 

For the CI workflow, calling the script directly makes sense. The makefile option would be for manual generation.

Reply:
We could add a target to call the script from the Makefile if you'd like, I don't have a strong preference. Just so I understand fully, is there a particular case you see the Makefile covering that manually triggering the workflow like what we have now for [Dependabot PRs](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/workflows/create-dependabot-pr.yml) wouldn't cover well?

Reply:
I'm also not sure if makefile is a good place for this. Makefile is usually meant for doing some stuff on the application not github

Reply:
Manually triggering the workflow works for me.